aliases.sh

```diff
@@ -18,3 +18,6 @@ alias git-global-ignore='git config --global core.excludesfile "~/utils/gitignor
 alias sf-cs-fix='php-cs-fixer fix --rules=@Symfony';
 
 alias gz='gzip --keep';
+
+alias textedit='open -e';
+alias gimp='open -a Gimp'
```

Comment:
Maybe something supporting [batch mode](https://www.gimp.org/tutorials/Basic_Batch/):

```suggestion
alias gimp='/Applications/GIMP.app/Contents/MacOS/gimp'
```